helm_deploy/values-dev.yaml

```diff
@@ -15,6 +15,8 @@ generic-service:
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api-dev.prison.service.justice.gov.uk"
     PRISONER_SEARCH_API_URL: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
     PRISON_API_URL: "https://api-dev.prison.service.justice.gov.uk"
+    VISIT_SCHEDULER_API_URL: "https://visit-scheduler-dev.hmpps.service.justice.gov.uk"
+    PRISONER_CONTACT_REGISTRY_API_URL: "https://visit-scheduler-dev.hmpps.service.justice.gov.uk"
```

Comment:
I know these are wrong, will wait until it's setup to correct, for now I've disabled the API calls